testsuite/single_node_performance.py

```diff
@@ -435,18 +441,23 @@ def print_table(
         number_of_threads_results = {}
 
         for execution_threads in EXECUTION_ONLY_NUMBER_OF_THREADS:
-            test_db_command = f"{BUILD_FOLDER}/aptos-executor-benchmark --execution-threads {execution_threads} {common_command_suffix} --skip-commit --blocks {NUM_BLOCKS_DETAILED}"
+            test_db_command = f"{BUILD_FOLDER}/aptos-executor-benchmark {PROFILE_FLAGS} --execution-threads {execution_threads} {common_command_suffix} --skip-commit --blocks {NUM_BLOCKS_DETAILED}``"
             output = execute_command(test_db_command)
 
             number_of_threads_results[execution_threads] = extract_run_results(
                 output, execution_only=True
             )
 
-        test_db_command = f"{BUILD_FOLDER}/aptos-executor-benchmark --execution-threads {NUMBER_OF_EXECUTION_THREADS} {common_command_suffix} --blocks {NUM_BLOCKS}"
+        test_db_command = f"{BUILD_FOLDER}/aptos-executor-benchmark {PROFILE_FLAGS} --execution-threads {NUMBER_OF_EXECUTION_THREADS} {common_command_suffix} --blocks {NUM_BLOCKS}"
         output = execute_command(test_db_command)
 
         single_node_result = extract_run_results(output, execution_only=False)
 
+        # Rename profile results
+        if os.environ.get("PROFILE"):
+            os.rename("profiling_results/profile.pb", f"profiling_results/{test.key.transaction_type}.pb")
+            os.rename("profiling_results/cpu_flamegraph.svg", f"profiling_results/{test.key.transaction_type}.svg")
+
```

Comment:
maybe print out in the end where to find profiling results, if they are accessible when run via CI workflow?